FamilyTree/po/it-local.po

```diff
@@ -19,291 +19,292 @@ msgstr ""
 
 #: FamilyTree/FamilyTree.gpr.py:3
 msgid "Family Tree"
-msgstr ""
+msgstr "Albero Genealogico"
 
 #: FamilyTree/FamilyTree.gpr.py:4
 msgid "Produces a graphical family tree."
-msgstr ""
+msgstr "Genera un albero genealogico grafico"
 
 #: FamilyTree/FamilyTree.py:102
 #, python-format
 msgid "Family Tree for %s"
-msgstr ""
+msgstr "Albero Genealogico per %s"
 
 #: FamilyTree/FamilyTree.py:668 FamilyTree/FamilyTree.py:673
 msgid "Unknown"
-msgstr ""
+msgstr "Sconosciuto"
 
 #: FamilyTree/FamilyTree.py:675
 #, python-format
 msgid "%(father)s and %(mother)s"
-msgstr ""
+msgstr "%(padre)s e %(madre)s"
```

Comment:
like for other replacement markers, please keep them as they are: "%(father) e %(mother)s"